docs/source-pytorch/common/lightning_module.rst

```diff
@@ -85,38 +85,43 @@ Here are the only required methods.
 .. code-block:: python
 
     import lightning.pytorch as pl
-    import torch.nn as nn
-    import torch.nn.functional as F
+    import torch
 
+    from lightning.pytorch.demos import Transformer
 
-    class LitModel(pl.LightningModule):
-        def __init__(self):
+
+    class LightningTransformer(pl.LightningModule):
+        def __init__(self, vocab_size):
             super().__init__()
-            self.l1 = nn.Linear(28 * 28, 10)
+            self.model = Transformer(vocab_size=vocab_size)
 
-        def forward(self, x):
-            return torch.relu(self.l1(x.view(x.size(0), -1)))
+        def forward(self, input, target):
+            return self.model(input, target)
 
         def training_step(self, batch, batch_idx):
-            x, y = batch
-            y_hat = self(x)
-            loss = F.cross_entropy(y_hat, y)
+            input, target = batch
```

Comment:
lets not use python keywords like `input`

Reply:
updated in [93a94fd](https://github.com/Lightning-AI/lightning/pull/18451/commits/93a94fd330b8da663797b727dc09293d37be46d5)